src/tools/genpolicy/rules.rego

```diff
@@ -232,6 +232,14 @@ allow_by_anno(p_oci, i_oci, p_storages, i_storages) {
     print("allow_by_anno 2: true")
 }
 
+check_namespace(p_namespace, i_namespace){
+    p_namespace == i_namespace
+}
+
+check_namespace(p_namespace, i_namespace){
+    p_namespace == ""
```

Comment:
My understanding is that we need something similar to:

check_namespace(p_namespace, i_namespace){
    p_namespace == i_namespace
}

check_namespace(p_namespace, i_namespace){
    p_namespace == ""
    add to state: i_namespace
}

check_namespace(p_namespace, i_namespace){
    data.namespace == i_namespace
}